stack.yaml

```diff
@@ -2,3 +2,5 @@ resolver: lts-14.27
 
 extra-deps:
   - http-directory-0.1.8
+  - git: https://github.com/juhp/koji-hs.git
+    commit: 84873619d266d31b4be2c1dc6eb24c0128e389dc
```

Comment:
Can you please add equivalent to cabal.project too so that the build may succeed in travis?

https://travis-ci.org/github/juhp/pkgtreediff/pull_requests

Reply:
That's good to know, thanks!